components/userstore-config/org.wso2.carbon.identity.user.store.configuration/src/main/java/org/wso2/carbon/identity/user/store/configuration/UserStoreConfigAdminService.java

```diff
@@ -134,6 +137,14 @@
                     originalPassword = userStoreProperties.get(JDBCRealmConstants.PASSWORD);
                     userStoreProperties.put(JDBCRealmConstants.PASSWORD, randomPhrase);
                 }
+
+                if (editedUserDTO != null && userStoreDTO.getDomainId().equals(editedUserDTO.getDomainId())) {
+                    for (PropertyDTO property : editedUserDTO.getProperties()) {
+                        userStoreProperties.put(property.getName(), property.getValue());
+                    }
+                    editedUserDTO = null;
```

Comment:
What is the need for resetting the editedUserDOT value ? If this is required, it's better to indicate it in a comment.